test/unit/org/apache/cassandra/utils/obs/BitUtilTest.java

```diff
@@ -0,0 +1,350 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.utils.obs;
+
+import org.apache.cassandra.utils.obs.BitUtil;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BitUtilTest {
+
+  @Test
+  public void testIntIsPowerOfTwo() {
+    Assert.assertEquals(true, BitUtil.isPowerOfTwo(2));
+  }
+
+  @Test
+  public void testLongIsPowerOfTwo() {
+    Assert.assertEquals(true, BitUtil.isPowerOfTwo(2L));
+  }
+
+  @Test
+  public void testLongNextHighestPowerOfTwo() {
+    Assert.assertEquals(0L, BitUtil.nextHighestPowerOfTwo(0L));
```

Comment:
0 covers the line but is this test meaningful enough?

Reply:
Other single tests that use 0 below could also be modified

Reply:
@paulkennethkent Done